packages/testlab/test/test-sandbox.unit.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/testlab
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {TestSandbox, expect} from '..';
+import {existsSync, readFile as readFile_} from 'fs';
+import {resolve} from 'path';
+import {createHash} from 'crypto';
+import * as util from 'util';
+const promisify = util.promisify || require('util.promisify/implementation');
+const rimraf = require('rimraf');
+const readFile = promisify(readFile_);
+
+describe('TestSandbox unit tests', () => {
+  let sandbox: TestSandbox;
+  let path: string;
+
+  beforeEach(createSandbox);
+  beforeEach(givenPath);
+  afterEach(deleteSandbox);
+
+  it('returns path of sandbox and it exists', () => {
+    expect(path).to.be.a.String();
+    expect(existsSync(path)).to.be.True();
+  });
+
+  it('creates a directory in the sandbox', async () => {
+    const dir = 'controllers';
+    await sandbox.mkdir(dir);
+    expect(existsSync(resolve(path, dir))).to.be.True();
+  });
+
+  it('copies a file to the sandbox', async () => {
+    const file = 'copy-me.js';
+    await sandbox.copy(resolve(__dirname, `fixtures/${file}`));
```

Comment:
```ts
await sandbox.copy(resolve(__dirname, 'fixtures', file));
```

As a long time Node.js user, I find it little bit unsettling to see `resolve` instead of `path.resolve`, but I guess it's just something that needs some time to get used to.